### DISCUSSION TITLE
Allow more "complicated shortcuts"

### DISCUSSION THREAD

**Discussion by hervehobbes** [adirh3/Fluent-Search#966] [Ideas]
Hello,

I'd like to be able to use some more "complicated" shortcuts, for example Ctrl + Win + Alt + P to show/call Fluent Search.
This is (a bit) like  Ctrl P in VS Code to open files but Ctrl + Win + Alt + P is a more "computer generic" shortcut to open/search for everything.

This also avoids creating conflicts with the keyboard shortcuts of certain applications

**Comment by adirh3** [comment]
This should be possible, when registering a hotkey type the keys one by one.
You can change the search hotkey in Settings -> Search -> Search hotkey

**Comment by hervehobbes** [comment]
Thanks again for your answer.
I did not succeed to set `Ctrl + Alt + Win + P` but I will stay with double Ctrl.

**Reply by adirh3** [reply]
That means it's already taken (or partially taken) so Windows can't register it

**Reply by hervehobbes** [reply]
I just tried with `Ctrl + Win + Alt + O`, that I verified before, and it's not running.
